### DISCUSSION TITLE
[React Aria Components] How to fit a column to its content?

### DISCUSSION THREAD

**Discussion by amirhhashemi** [adobe/react-spectrum#5809] [Q&A] [answered]
There are CSS tricks to fit a native table's column to its content. For example, see [this stackoverflow answer](https://stackoverflow.com/a/11267268/16225339).

When using `ResizableTableContainer`, the column width is calculated with Javascript and therefor, those tricks won't work. 

What's the best way to fit a column's width to its content when using `ResizableTableContainer`? One possible solution is to get the column's width with Javascript and set the width manually. But I want to know if there are better ways.

**Comment by snowystinger** [comment, marked as answer]
Thanks for the discussion!
Unfortunately, that's not a feature that ResizableTableContainer supports at this time. We could consider adding it and would be willing to consider a contribution. In the meantime, the recommended approach would be to set it yourself, as you noted.
